src/components/Error.jsx

```diff
@@ -131,10 +137,22 @@ export const BZReportModal = ({
         PRETTY_NAME: prettyName,
     } = useContext(OsReleaseContext);
 
+    // anaconda version is also used on AboutModalVersions
+    // should we add it to a context?
```

Comment:
another argument for possibly adding this info to context: when loading up really fast the data might not be loaded in time to be added to BZ template. At least I saw this issue locally